python/lsst/analysis/tools/contexts/_baseContext.py

```diff
@@ -29,7 +29,7 @@
 from functools import partial, update_wrapper
 from typing import TYPE_CHECKING, Callable, Iterable, Union, cast, overload
 
-from lsst.pipe.tasks.configurableActions import ConfigurableActionStruct
+from lsst.pex.config.configurableActions import ConfigurableActionStruct
 
 if TYPE_CHECKING:
     from ..interfaces import AnalysisAction
```

Comment:
A further mention of `pipe_tasks` is also made in this file on L116 (leaving a comment here as it's as close as I can get). This may want to be updated also.